### ISSUE TITLE
[account] how account type affect report broken

### ISSUE THREAD

**Issue by jeffery9** [odoo/odoo#27764]
Impacted versions:
12

Steps to reproduce:
setup a new account, found that  how account type affect your report is broken, do not like previous edition any more.

Current behavior:

Expected behavior:
side by side should be better for reading.

Video/Screenshot link (optional):

![image](https://user-images.githubusercontent.com/1404460/46915953-1e257880-cfe6-11e8-9427-a5a6e9223fdb.png)

**Comment by pedrobaeza** [comment]
I don't get the problem. Can you rephrase it for trying to understand?

**Comment by jeffery9** [comment]
the same issue on runbot.

![image](https://user-images.githubusercontent.com/1404460/46916664-3601fa00-cff0-11e8-85c6-e15649e0731e.png)

**Comment by sswapnesh** [comment]
Hello @jeffery9 
Do you mean it should look like this screenshot (Taken from v11)?
![screenshot from 2018-10-14 22-57-10](https://user-images.githubusercontent.com/16624719/46919943-98fd8c00-d004-11e8-8ae8-2bb4487816a1.png)

**Comment by sswapnesh** [comment]
Here is another Related Bug for v11/12

```
<div style="position: relative; width: 250px; height: 277px;" id="layer">
               <div style="position: relative; left: 0px; top: 0px; visibility: visible; width: 250px; height: 277px;">               <svg pointer-events="none" overflow="auto" width="712.5" height="265" viewBox="-4 -4 712.5 265" style="position: relative; top: -4px; left: -4px; overflow: auto; visibility: visible; width: 400.5px; height: 265px;">
                                                       <defs>
                                                         <marker id="arrow" markerWidth="70" markerHeight="70" refx="10" refy="6" orient="auto">
                                                             <path d="M2,1 L2,12 L10,6 L2,1" stroke="rgb(173,216,230)" fill="rgb(173,216,230)"/>
                                                         </marker>
                                                       </defs>
                                                       <path pointer-events="visiblePainted" shape-rendering="auto" d="M 150 -110 q 270 100 150 150 t 160 120" fill="none" stroke-dasharray="8,3" stroke="rgb(173,216,230)" stroke-linecap="butt" stroke-width="4" stroke-opacity="1" stroke-linejoin="round" transform="matrix(1,0,0,1,0,0)" fill-opacity="1" style="marker-end: url(#arrow);">
                                                       </path>
                                                     </svg>
                                                   </div>
                                               </div>
```

This Code is present in v11/12 but doesn't not make any line on view.
![screenshot 9](https://user-images.githubusercontent.com/16624719/46920126-29d56700-d007-11e8-8b0e-9dcdf4ea3b6f.png)

**Comment by C3POdoo** [comment]
Dear @jeffery9,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
